proto/irismod/token/v1/tx.proto

```diff
@@ -30,14 +30,24 @@ service Msg {
   rpc TransferTokenOwner(MsgTransferTokenOwner)
       returns (MsgTransferTokenOwnerResponse);
 
-  // SwapFeeToken defines a method for swap some fee token
+  // SwapFeeToken defines a method for swaping some fee token
   rpc SwapFeeToken(MsgSwapFeeToken) returns (MsgSwapFeeTokenResponse);
 
+  // SwapToERC20 defines a method for swaping some erc20 token
+  rpc SwapToERC20(MsgSwapToERC20) returns (MsgSwapToERC20Response);
+
+  // SwapFromERC20 defines a method for some native token
+  rpc SwapFromERC20(MsgSwapFromERC20) returns (MsgSwapFromERC20Response);
+
   // UpdateParams defines a governance operation for updating the token
   // module parameters. The authority is defined in the keeper.
   //
   // Since: cosmos-sdk 0.47
   rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
+
+  // DeployERC20 defines a governance operation for deploying erc20 token for
+  // the native token
```

Comment:
.. defines a governance operation for deploying an ERC20 contract that binds to a native token